ramls/item.json

```diff
@@ -231,7 +231,8 @@
             "On order",
             "Paged",
             "Declared lost",
-            "Order closed"
+            "Order closed",
+            "Claim returned"
```

Comment:
Based upon scenario 6, my interpretation is that the item status becomes `Claimed returned` and there needs to be a date recorded for when this last occurred.

Please can you follow up with Emma about the requirements for this?

Reply:
Thanks, good point. Posted a comment.

Reply:
Changed to `Claimed returned`. The date and time when item was last marked as claimed returned seems is not required for now. Date and time when loan has been marked as claimed returned will be on loan record.